src/handlers/input-type.ts

```diff
@@ -138,7 +138,7 @@ export function inputType(
       // TODO: Deprecated and should be removed
       importDeclarations.add('Decimal', '@prisma/client/runtime/library');
```

Comment:
This default `@prisma/client` is still here. So, this is partial fix, will be working if are you not using `Decimal`

Reply:
any solution to this?

Reply:
@unlight  I will fix this right now!  Sorry about that, I was initially under the impression that it did not generate the runtime types as well, only the base of the package...

Reply:
@unlight I just resolved this and it should be good now!